graphs/graph_modules/play_count_by_month_graph.py

```
@@ -0,0 +1,268 @@
+            if "data" in serie:
+                if not isinstance(serie["data"], (list, tuple)):
+                    errors.append(f"Series {idx} data is not a list")
+                elif len(serie["data"]) != len(months):
+                    errors.append(
+                        f"Series {idx} ({serie.get('name', 'unnamed')}) data length mismatch: "
+                        f"expected {len(months)}, got {len(serie['data'])}"
+                    )
+                    
+            # Validate data types if data exists
+            if "data" in serie and isinstance(serie["data"], (list, tuple)):
+                if not all(isinstance(x, (int, float)) for x in serie["data"]):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)